src/download-header/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useRef, useEffect, Fragment } from 'react';
+
+const DownloadHeader = ({ model, licenceType, isGranted, basename, showWord = true, showPdf = true }) => {
+  const [modalShowing, updateModalShowing] = useState(false);
+  const container = useRef(null);
+  const download = useRef(null);
+  let title = 'Licence';
+
+  switch (licenceType) {
+    case 'ppl':
+      title = model.title || 'Untitled project';
+      break;
+    case 'pil':
+      title = 'Personal licence';
+      break;
+    case 'pel':
+      title = 'Establishment licence';
```

Comment:
Is this in the designs? It feels weirdly inconsistent to use the project title for projects, but not the establishment name for establishments?

Reply:
Yup.